Components/DB/DBConfigurableConsumer.php

```diff
@@ -131,9 +127,13 @@ public function consume(EndpointInterface $endpoint)
                 $endpoint->handle($message);
 
                 if ($this->logger) {
+                    $microTime = number_format(microtime(true), 6, '.', '');
```

Comment:
Why not create a method on Abstract and calls here?
Like consumerLogMessage();
The code is the same as for the message.

Reply:
Good idea